ios/FluentUI/Vnext/Button/Button.swift

```diff
@@ -158,12 +181,37 @@ public struct MSFButtonView: View {
         })))
         buttonView.tokens.windowProvider = self
         view.backgroundColor = UIColor.clear
+        setupLargeContentViewer()
```

Comment:
may be shouldEnableLargeContentViewerForOSVersion should be called before setting up largecontentviewer?

Reply:
The constant **shouldEnableLargeContentViewerForOSVersion** is initialized when the class is initialized, which happens before we call shouldEnableLargeContentViewerForOSVersion occurs.